app/assets/stylesheets/variables.scss

```diff
@@ -22,6 +21,9 @@ $tan: #fdf9f3;
 $off-white: #fffffe;
 $bold-blue: #0045ff;
 
+$max-width: 1250px;
+$header-height: 46px; // todo: should be calculated automatically based on content
```

Comment:
What if we rounded this up to `48px`? It's a nice value that mirrors our divisible-by-4 interest.

Reply:
As my comment says in the code: I hope we could get rid of that at all in future PR so the height could be calculated automatically based on its content. Right now, changing that value would affect visuals too much - I prefer not to do it in this PR.

But one of the next PRs will be about actually making header look more like in North Star project, and I'll try to address it then.

Reply:
👍🏽